spec/services/diaper_partner_client_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'spec_helper'
+require_relative '../support/env_helper'
```

Comment:
I don't think you need this -- cf. `/spec/rails_helper.rb#32`

```ruby
Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
```
If you put a ruby file under the `spec/support` dir, it will be pulled in for specs.

Reply:
Oh nice. Happy to remove those lines in a subsequent PR if you'd like